widget/playground/src/containers/SupportedTokens/SupportedTokens.tsx

```diff
@@ -0,0 +1,70 @@
+import type { Type } from '../../types';
+
+import { ChainsIcon } from '@rango-dev/ui';
+import React from 'react';
+
+import { MultiSelect } from '../../components/MultiSelect/MultiSelect';
+import { useConfigStore } from '../../store/config';
+import { useMetaStore } from '../../store/meta';
+import { tokensAreEqual } from '../../utils/common';
+
+export function SupportedTokens({ type }: { type: Type }) {
```

Comment:
agreed.

Reply:
I think we could hide this section for now in playground until we fix problem related config tokens structure and exclude option. In current state, in most usecases, it's very laggy/slow because of huge number of tokens exported.
I'm not sure the only problem for being laggy is related to this or not, maybe some filters/some combinations could be handled in another way if they are the reason. We could check it after config changes.